src/main/java/space/obminyashka/items_exchange/service/impl/LocationServiceImpl.java

```diff
@@ -123,6 +136,53 @@ public String createParsedLocationsFile(List<RawLocation> creatingData) throws I
         return Files.readString(Path.of(locationInitFilePath).toAbsolutePath(), StandardCharsets.UTF_8);
     }
 
+    @Override
+    public String createParsedAreasFile(List<RawLocation> creatingData) throws IOException {
+        List<Area> areas = mapCreatingDataToAreas(creatingData);
+        String initString = "INSERT INTO area (id, name_ua, name_en) VALUES";
+        try (BufferedWriter writer = Files.newBufferedWriter(Path.of(areaInitFilePath), StandardCharsets.UTF_8)) {
+            writer.append(initString);
+            for (int i = 0; i < areas.size(); i++) {
+                Area area = areas.get(i);
+                writer.append(areaToFormatterString(area));
+                if (i < areas.size() - 1) writer.append(",");
+            }
+        }
+        return Files.readString(Path.of(areaInitFilePath).toAbsolutePath(), StandardCharsets.UTF_8);
+    }
+
+    @Override
+    public String createParsedDistrictsFile(List<RawLocation> creatingData) throws IOException {
+        List<District> districts = mapCreatingDataToDistricts(creatingData);
+        String initString = "INSERT INTO district (id, area_id, name_ua, name_en) VALUES";
+
+        try (BufferedWriter writer = Files.newBufferedWriter(Path.of(districtInitFilePath), StandardCharsets.UTF_8)) {
+            writer.append(initString);
+            for (int i = 0; i < districts.size(); i++) {
+                District district = districts.get(i);
+                writer.append(districtToFormatterString(district));
+                if (i < districts.size() - 1) writer.append(",");
+            }
+        }
+        return Files.readString(Path.of(districtInitFilePath), StandardCharsets.UTF_8);
+    }
+
+    @Override
+    public String createParsedCityFile(List<RawLocation> creatingData) throws IOException {
+        List<City> cities = mapCreatingDataToCity(creatingData);
+        String initString = "INSERT INTO city (id, district_id, name_ua, name_en) VALUES";
+
+        try (BufferedWriter writer = Files.newBufferedWriter(Path.of(cityInitFilePath), StandardCharsets.UTF_8)) {
+            writer.append(initString);
+            for (int i = 0; i < cities.size(); i++) {
+                City city = cities.get(i);
+                writer.append(cityToFormatterString(city));
+                if (i < cities.size() - 1) writer.append(",");
+            }
+        }
+        return Files.readString(Path.of(cityInitFilePath), StandardCharsets.UTF_8);
+    }
+
```

Comment:
You can create new generic method which has arguments like path name, type of class etc. instead of 3 new

Reply:
done